### PR TITLE
Fix Slider Within Slider

### PR DESCRIPTION
**Pull request by imranhsayed**
Fixes [Slider within slider Issue](https://github.com/Travelopia/web-components/issues/19)

### CODE REVIEW

File: src/slider/index.html [hunk 1]
```diff
@@ -128,6 +128,37 @@
 			</tp-slider-nav>
 			<tp-slider-count current="1" total="3" format="$current / $total">1 / 3</tp-slider-count>
 		</tp-slider>
+
+		<!--Slider within slider-->
```
Comment (junaidbhura) [inline src/slider/index.html:132]: @imranhsayed Lets say `Nested sliders`

File: src/slider/style.scss [hunk 2]
```diff
@@ -37,25 +37,25 @@ tp-slider-nav {
 // Fade in Styles.
 tp-slider[behaviour="fade"] {
 
-	tp-slider-slides {
+	> tp-slider-track > tp-slider-slides {
```
Comment (junaidbhura) [inline src/slider/style.scss:40]: @imranhsayed Not sure about `> tp-slider-track` - any other solution? Do we need to make this update at all?

File: src/slider/tp-slider-count.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { TPSliderElement } from './tp-slider';
```
Comment (junaidbhura) [inline src/slider/tp-slider-count.ts:1]: @imranhsayed Lets add the DocBlock

**Review by junaidbhura — CHANGES_REQUESTED** [review]

File: src/slider/style.scss (hunk 2 above)
Comment (imranhsayed) [reply src/slider/style.scss:40]: @junaidbhura 
If we don't use the direct child then it adds those styles to nested slider slides also, while it should be only adding it to the parent slider.

Comment (junaidbhura) [reply src/slider/style.scss:40]: @imranhsayed Shouldn't this be applied to everything anwyway?

Comment (imranhsayed) [reply src/slider/style.scss:40]: No @junaidbhura It breaks the inner slider styles without this.

Comment (imranhsayed) [reply src/slider/style.scss]: @junaidbhura As discussed, I have made the updates.

**Review by junaidbhura — APPROVED** [review]

**Pull request merged**